_episodes/01-intro.md

```diff
@@ -59,98 +59,56 @@ loops back and waits for you to enter another command.
-Flags start with a single dash (`-`) or two dashes (`--`), and change the behaviour of a command.
-Arguments tell the command what to operate on (e.g. files and directories).
-Sometimes options and arguments are referred to as **parameters**.
-A command can be called with more than one option and more than one argument: but a
-command doesn't always require an argument or an option.
-
-In the second line of the example above, our **command** is `ls`, with an **option** `-F` and an
-**argument** `/`. Each part is separated by spaces: if you omit the space
-between `ls` and `-F` the shell will look for a command called `ls-F`, which
-doesn't exist. Also, capitalization matters: `LS` is different from `ls`.
-
-Next we see the output that our command produced. In this case it is a listing
-of files and folders in a location called `/` - we'll cover what all these mean
-later today. Those using a macOS might recognize the output in this example.
-
-Finally, the shell again prints the prompt and waits for you to type the next
-command.
-
-In the examples for this lesson, we'll show the prompt as `$ `. You can make your
-prompt look the same by executing the command `PS1='$ '`. But you can also leave
-your prompt as it is - often the prompt includes useful information about who and where
-you are.
-
-Open a shell window and try executing `ls -F /` for yourself (don't forget that spaces
-and capitalization are important!). You can change the prompt too, if you like.
+So let's try our first command, which will list the contents of the current directory:
 
-### How does the shell know what `ls` and its options mean?
-
-Every command is a program stored somewhere on the computer, and the shell keeps a
-list of places to search for commands (the list is in a **variable** called `PATH`,
-but those are concepts we'll meet later and are not too important at the moment). Recall
-that commands, options and arguments are separated by spaces.
-
-So let's look at the REPL (read-evaluate-print loop) in more detail. Notice that the
-"evaluate" step is made of two parts:
-
-1. Read what was typed (`ls -F /` in our example)
-    The shell uses the spaces to split the line into the command, options, and arguments
-2. Evaluate:
-    a. Find a program called `ls`
-    b. Execute it, passing it the options and arguments (`-F` and `/`) to
-       interpret as the program sees fit
-3. Print the output produced by the program
+~~~
+$ ls
+~~~
+{: .language-bash}
 
-and then print the prompt and wait for you to enter another command.
+~~~
+Desktop     Downloads   Movies      Pictures
+Documents   Library     Music       Public
+~~~
+{: .output}
 
 > ## Command not found
 > If the shell can't find a program whose name is the command you typed, it
-> will print an error message like:
+> will print an error message such as:
 >
 > ~~~
-> $ ls-F
+> $ ks
 > ~~~
 > {: .language-bash}
 > ~~~
-> -bash: ls-F: command not found
+> ks: command not found
```

Comment:
Depending on the operating system, the command-not-found bash hook might suggest a program to install to supply ks....